autoload/vsnip/snippet/node/placeholder.vim

```diff
@@ -21,6 +21,7 @@ function! s:Placeholder.new(ast) abort
   \   'follower': v:false,
   \   'choice': get(a:ast, 'choice', []),
   \   'children': vsnip#snippet#node#create_from_ast(get(a:ast, 'children', [])),
+  \   'transform': get(a:ast, 'transform', v:null),
```

Comment:
I think the transform should be new node type.

Reply:
👍🏼 Sounds good! I'll look into it.

Reply:
@hrsh7th Done!

Let me know if something needs to be refactored.